h/static/styles/thread.scss

```diff
@@ -15,6 +15,26 @@ $thread-padding: $annotation-card-left-padding;
   }
 }
 
+.annotation-unavailable-message {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid $gray-lighter;
+  padding-top: 30px;
```

Comment:
TODO: The multiples of 10px here are chosen to align with the 10px grid which the new design is based off. Now would be a good time to introduce that as a variable in `variables.scss`